index.html

```diff
@@ -179,5 +179,37 @@ <h3>Multi-Post Stories</h3>
       </div>
     </div>
 </section>
+<section class="contact-sec">
+  <h3> CONTACT US  </h3>
+   <p>If you have an application you are interested in developing, 
+    a feature that you need built or a project that needs coding. I’d love to help with it 
+    </p>
+<form class="contact-form" action="https://formspree.io/f/mknyzlre" method="post">
+<input type="text" name="username" placeholder="Text Input" maxlength="30" required>
+<input type="email" name="email" placeholder="jeremy@gmail.com" maxlength="30" required>
+<textarea name="text" placeholder="Enter text here" maxlength="500" required>
+</textarea>
+
+<div class="btn">
+  <button type="submit">See Project</button>
+</div>
+</form>
```

Comment:
Your form is working like a charm 💪😎 Well done

Reply:
HTML validations are perfectly setup ✔

Reply:
Thanks for the review 👍🏽